packages/cli/src/constructs/check.ts

```diff
@@ -65,11 +66,11 @@ export interface CheckProps {
   frequency?: number
   environmentVariables?: Array<EnvironmentVariable>
   /**
-   * The id of the check group this check is part of.
+   * The id of the check group this check is part of. Set this by calling `someGroup.ref()`
    */
   groupId?: Ref
   /**
-   * List of alert channel subscriptions.
+   * List of alert channels this check should subscribe to.
```

Comment:
Nit - It doesn't seem to me like checks are subscribed to alert channels. The alert channel gets notified when the check changes, rather than the other way around.

Maybe "List of alert channels to notify when the check fails or recovers"